workshops/memory game/README.md

```diff
@@ -0,0 +1,376 @@
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    },
+    {
+      name: '1',
+      img: 'https://cloud-5ystxzer7.vercel.app/11.png'
+    },
+    {
+      name: '2',
+      img: 'https://cloud-5ystxzer7.vercel.app/22.png'
+    },
+    {
+      name: '3',
+      img: 'https://cloud-5ystxzer7.vercel.app/33.png'
+    },
+    {
+      name: '4',
+      img: 'https://cloud-5ystxzer7.vercel.app/44.png'
+    },
+    {
+      name: '5',
+      img: 'https://cloud-5ystxzer7.vercel.app/55.png'
+    },
+    {
+      name: '6',
+      img: 'https://cloud-5ystxzer7.vercel.app/06.png'
+    }
+  ]
+})
+```
+
+Now we are going to creating our game board.
+
+## Game Board
+
+Pick up the `div` element with `class = "board"` using [querySelector](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) and define it as `board`. Create a function `createBoard()` and loop over through elements in `cardArray` using `for` loop and add cards to our game board.
+
+Create a constant for our placeholder image. We are going to use it many times in the project.
+
+```javascript
+const placeholder = 'https://cloud-5ystxzer7.vercel.app/7placeholder.png'
+```
+
+Create a `img` element using `document.createElement` for the every card. Using `setAttribute()`, add `src` and `data-id` attributes to the image. Let's first set the `src` attribute to the placeholder image (i.e, let's assume the placeholder image as reversed card). The link to the placeholder image is given in the code, otherwise add the relative path to the image.
+
+![place holder image](https://cloud-5ystxzer7.vercel.app/7placeholder.png)
+
+Here, [createElement()](https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement) creates the HTML element specified by tagName, [setAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute) sets the value of an attribute on the specified element and [data attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) allow us to store extra information on standard in the HTML tag. According to the game, we have to flip the card that is clicked. So add a `click` event-listener for the card. Every time a card get clicked, `flipCard` function will be excuted, which we will define later on the flow. Then add the `card` into the `board` using `appendChild()`. The method [appenChild()](https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild) adds a node to the end of the list of children of a specified parent node.
+
+```javascript
+const board = document.querySelector('.board')
+const placeholder = 'https://cloud-5ystxzer7.vercel.app/7placeholder.png'
+
+// create game board
+function createBoard() {
+  for (let i = 0; i < cardArray.length; i++) {
+    var card = document.createElement('img')
+    card.setAttribute('src', placeholder)
+    card.setAttribute('data-id', i)
+    //card.addEventListener('click', flipCard)
```

Comment:
Kind of unclear to me why this event listener is commented if the instructions before this code block stated to make the event listener.